docs/source/user/interface.rst

```diff
@@ -63,6 +63,9 @@ and a list of tabs in the main work area:
    :align: center
    :class: jp-screenshot
 
+The icons and labels column that allows to switch between tabs is called
+Activity Bar in JupyterLab.
```

Comment:
```suggestion
The icons and labels column that enables switching between sidebar activities is called
Activity Bar.
```

Reply:
It looks like patch from this suggestion got lost during rebase.